installer/conf/telegraf.conf

```diff
@@ -532,7 +532,8 @@
   name_prefix="container.azm.ms/"
   ## An array of urls to scrape metrics from.
   urls = ["$CADVISOR_METRICS_URL"]
-  fieldpass = ["kubelet_docker_operations", "kubelet_docker_operations_errors"]
+  ## Include "$KUBELET_RUNTIME_OPERATIONS_TOTAL_METRIC", "$KUBELET_RUNTIME_OPERATIONS_ERRORS_TOTAL_METRIC" when we add for support for 1.18
+  fieldpass = ["$KUBELET_RUNTIME_OPERATIONS_METRIC", "$KUBELET_RUNTIME_OPERATIONS_ERRORS_METRIC"]
```

Comment:
I see that you don't set these env for non docker clusters in oms-docker PR. Which means we will be passing everything (as pass list is empty) - meaning we will collect all cadvisor metrics exposed @ /metrics.

Reply:
as discussed, default will be set these.
export KUBELET_RUNTIME_OPERATIONS_METRIC="kubelet_docker_operations"
export KUBELET_RUNTIME_OPERATIONS_ERRORS_METRIC="kubelet_docker_operations_errors"